sql/2020/09_Performance/web_vitals_distribution_by_device.sql

```diff
@@ -0,0 +1,183 @@
+#standardSQL
+# WebVitals distribution by device
+
+WITH
+  base AS (
+  SELECT
+    device,
+
+    fast_fid,
+    avg_fid,
+    slow_fid,
+    
+    fast_lcp,
+    avg_lcp,
+    slow_lcp,
+    
+    small_cls,
+    medium_cls,
+    large_cls,
+
+    fast_fcp,
+    avg_fcp,
+    slow_fcp,
+
+    fast_ttfb,
+    avg_ttfb,
+    slow_ttfb,
+    
+  FROM
+    `chrome-ux-report.materialized.device_summary`
+  WHERE
+    device IN ('desktop','phone')
+    AND date >= '2020-09-01'
+    AND date <= "2020-09-30"
```

Comment:
```suggestion
    AND date = '2020-08-01'
```

Reply:
My expectation here was to use the correct September data.
The query is indeed failing, but only until the data becomes available.

Reply:
Only Lighthouse stats were wrong for August. CrUX data was find for August so can use that. It does create a slight mis-match but we're either going to mismatch with rest of Almanac if we use September for everything so agree with Rick that we should use August where possible.

Reply:
Rolled back all CrUX table queries